data/json/bionics.json

```diff
@@ -41,6 +41,15 @@
     "mutation_conflicts": [ "LEG_TENTACLES", "LEG_TENT_BRACE", "GASTROPOD_FOOT", "GASTROPOD_BALANCE", "AMORPHOUS" ],
     "flags": [ "BIONIC_FAULTY" ]
   },
+  {
+    "id": "bio_ar",
+    "type": "bionic",
+    "name": { "str": "Integrated AR System" },
+    "description": "Provide user with augmented reality overlay allowing the user to zoom, and it displays handy information about local topology.",
+    "occupied_bodyparts": [ [ "eyes", 1 ] ],
+    "flags": [ "USES_BIONIC_POWER" ],
```

Comment:
You are missing `"mutation_conflicts"` for all the eye mutations.

Reply:
Should it tho? I can see it conflicting with eyestalks, but somthing like lizard eyes seems fine

Reply:
I think I'm going to flavour it like lenses that go over you're eyes. I'll tweak the description to reflect that and set the list of conflict similar to protective lenses